desktop/assets/texts/zh-CN/settings.ftl

```diff
@@ -18,7 +18,7 @@ open-url-mode-deny = 拒绝
 load-behavior = 加载方式
 load-behavior-streaming = 流
 load-behavior-delayed = 延迟
-load-behavior-blocking = 阻止
+load-behavior-blocking = 堵塞
```

Comment:
According to machine brain:

**阻止 (zǔzhǐ)**

> Meaning: To prevent or stop someone or something from doing something. It emphasizes intervention to halt an action or process.
> Usage: Usually applies to actions, behaviors, or events and is often intentional.

**堵塞 (dǔsè)**

> Meaning: To block, clog, or obstruct, often referring to physical or metaphorical blockages that cause a lack of flow or progress.
> Usage: Commonly describes traffic jams, clogged pipes, or situations where movement is physically or figuratively impeded.

So in this case I think the latter is indeed more fitting.